src/App.tsx

```diff
@@ -1,18 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
+const getVisibleItems = (perPage: number, currentPage: number) => {
+  const lastIndex = perPage * currentPage;
+
+  return items.slice(lastIndex - perPage, lastIndex);
+};
+
 export const App: React.FC = () => {
+  const [currPage, setCurrPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+  const [total] = useState(items.length);
```

Comment:
why do we need to save this value to state? 
more logical to just save it co constant and reuse

Reply:
I'm still sure you don't need this state